src/nvim/os/process.c

```diff
@@ -71,24 +72,30 @@ static bool os_proc_tree_kill_rec(HANDLE process, int sig)
   return (bool)TerminateProcess(process, (unsigned int)sig);
 }
 /// Kills process `pid` and its descendants recursively.
-bool os_proc_tree_kill(int pid, int sig)
+bool os_proc_tree_kill(Process *proc, int sig)
 {
```

Comment:
This was int in case we want to add a kill() user function later. For arbitrary processes.

Reply:
Ok.